DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: mice
 Type: Package
-Version: 3.14.9
+Version: 3.14.10
```

Comment:
Do not put version changes in DESCRIPTION since we may wish to collect multiple PR's within one version change

Reply:
did revert change in version in the PR